app_integrations/apps/gsuite.py

```diff
@@ -156,6 +169,11 @@ def keyfile_validator(keyfile):
                     'description': ('the path on disk to the JSON formatted Google '
                                     'service account private key file'),
                     'format': keyfile_validator
+                },
+            'delegation_email':
+                {
+                    'description': 'the service account user email to delegate access to',
+                    'format': re.compile(r'^[A-Za-z0-9-_.]+@[A-Za-z0-9-.]+\.[A-Za-z]{2,}$')
```

Comment:
Maybe change this to allow also `+`, so emails like `foo+bar@foobar.com` works. It should be just adding the character to the first grouping.